### DISCUSSION TITLE
팀코톡 주제

### DISCUSSION THREAD

**Discussion by Byeongju-Kong** [woowacourse-teams/2022-sokdak#463] [Ideas]
팀코톡으로 공유하면 좋을 것 같은 주제를 달아주세요~!

**Comment by hyunrrr** [comment]
- 운영체제 관련
    - 프로세스, 스레드
    - 스케쥴링

- 네트워크
    - 브라우저에 www.google.com을 치면 어떻게 플로우가 흘러가는가

**Comment by Byeongju-Kong** [comment]
- 프로세스 vs 스레드
- 웹 동작 방식
- 캐싱

**Comment by BETTERFUTURE4** [comment]
- 소나큐브/클라우드 세팅 과정
